quinn-proto/src/connection/mod.rs

```diff
@@ -2313,6 +2313,7 @@ where
         let is_0rtt = self.spaces[SpaceId::Data].crypto.is_none();
         let mut is_probing_packet = true;
         let mut close = None;
+        let payload_len = payload.len();
```

Comment:
I'd prefer to just squash this commit into the previous one, as this one is not too complex, and it makes it more obvious what's going on.

Reply:
OK, I squashed the two commits into one.